apps/sms/services/js/message_manager.js

```diff
@@ -41,28 +54,54 @@ var MessageManager = {
     );
   },
 
+  /**
+   * Event handler executed when a message is about to be 
+   * sent.
+   * @param {object} e The event object.
```

Comment:
__nit:__ If method isn't used anywhere else except for this object - then please mark such methods with ```@private```